ext/ips/puppetagent.xml

```diff
@@ -34,7 +34,7 @@
       </common_name>
       <documentation>
         <manpage title="puppet" section="1"/>
-        <doc_link name="puppetlabs.com" uri="http://puppetlabs.com/puppet/introduction"/>
+        <doc_link name="puppetlabs.com" uri="https://puppetlabs.com/puppet/introduction"/>
```

Comment:
This redirects to https://puppetlabs.com/puppet/what-is-puppet - is that where we want the link to go @nfagerlund ?